int-tests/006-tunnel.t

```diff
@@ -0,0 +1,18 @@
+Create a group to play around with.
+  $ GROUP=002-tunnel-group
+  $ mkdir $GROUP
+  $ printf "[Unit]\nDescription=$GROUP Inago Update Test Unit\n\n[Service]\nExecStart=/bin/bash -c \"while true; do echo Hi; sleep 10; done\"\n" > $GROUP/$GROUP-foo@.service
+  $ printf "[Unit]\nDescription=$GROUP Inago Update Test Unit\n\n[Service]\nExecStart=/bin/bash -c \"while true; do echo Hi; sleep 10; done\"\n" > $GROUP/$GROUP-bar@.service
+
+Validate the test group.
+  $ inagoctl validate $GROUP
+  Group '002-tunnel-group' is valid.
+  Groups are valid globally.
+
+Submit 2 slices of update group, using the tunnel flag.
+  $ inagoctl --tunnel=${INAGO_TUNNEL_ENDPOINT} --ssh-strict-host-key-checking=false submit $GROUP 2
```

Comment:
Since I had problems when using the tunnel during updates, we should also test the update command here.